bolt.rb

```diff
@@ -41,13 +59,38 @@ def list
 
 class Target < Wash::Entry
   label 'target'
-  state :connection_info
+  parent_of VOLUMEFS
+
+  def known_hosts(host_key_check)
+    return nil unless host_key_check == false
+
+    # Disable host key checking by redirecting known hosts to an empty file
+    Gem.win_platform? ? 'NUL' : '/dev/null'
```

Comment:
Wash doesn't work with Windows so why is this here? A comment would help.

Reply:
Was pretty much to avoid a future issue when it does.